src/modules/players.haml

```diff
@@ -0,0 +1,86 @@
+.container
+    .jumbotron.subhead
+        %h1
+            = @page_title = "Modules"
+            %p.lead Game Settings
+    %section
+        .row
+            = partial('modules_sidebar')
+            .col-sm-9
+                .page-header
+                    %h1= @page_sub_title = "Players"
+                :markdown
+                    The players module is the basis for all team-less gamemodes, such as FFA, UHC, or hunger games style matches.
+                    This module is not limited to matches focused on combat,
+                    it can also be used in combat-less game modes where players have to collect specific items, etc.
+                    Win conditions are set by using the score and time limit modules.
+
+                    `NOTE:` This gamemode is not compatible with the teams module!
+
+                .table-responsive
+                    %table.table.table-striped.table-condensed
+                        %thead
+                            %tr
+                                %th Element
+                                %th Description
+                                %th
+                        %tbody
+                            %tr
+                                %td
+                                    %pre
+                                        %code= html_escape('<players/>')
+                                %td The players node, containing player settings.
+                                %td
+
+                %h5 Players Attributes
+                .table-responsive
+                    %table.table.table-striped.table-condensed
+                        %thead
+                            %tr
+                                %th{:style => "min-width: 130px;"}  Attribute
+                                %th Description
+                                %th Value
+                        %tbody
+                            %tr
+                                %td
+                                    %code min
+                                %td
+                                    Minimum amount of players required.
+                                %td
+                                    %span.label.label-primary Number
+                            %tr
+                                %td
+                                    %code max
+                                %td
+                                    Maximum amount of players. (soft limit)
```

Comment:
Let's fix this before making another copy of it. This one can just say "Maximum number of players that can play in the match."